docs/java_tutorials.md

```diff
@@ -1,5 +1,12 @@
 # Tutorials for Java Users
 
-TBF
+Apart from the Java ``PIE`` interface exposed by ``GRAPE-jdk``, efforts have also been made to 
+integrate Graphscope with some famous Graph computing systems, like Apache Giraph, Apache GraphX. 
+So not only do the users run java algorithms developed with ``GRAPE-jdk`` on GraphScope, 
+but also they can run existing graph algorithms developed for Giraph or GraphX on GraphScope.
 
-( this document describes how to use GraphScope Java API, or its headers with libs.)
\ No newline at end of file
+Follow the tutorials below, give it a shot on trying exploring GraphScope Java.
+
+- [Querying graph via GraphScope JavaSDK](https://graphscope.io/docs/latest/java-tutorial-0-pie.html)
```

Comment:
- Please use relative URLs, otherwise I cannot find these files in the preview or other deployments. (e.g., intranet in the future. )
- Please use the filename as unified underline `_` rather than `-`.